src/main/java/io/apiapp/springbootstarter/tvprogramme/TVProgramme.java

```diff
@@ -0,0 +1,73 @@
+package io.apiapp.springbootstarter.tvprogramme;
+
+
+public class TVProgramme {
```

Comment:
Instead of having class with set methods, mark every field as final and pass all the parameters via constructor like you do right now. Remove empty constructor.

more reading: https://www.baeldung.com/java-immutable-object